gateway-rest/src/main/java/io/camunda/zeebe/gateway/rest/ZeebeRestController.java

```diff
@@ -0,0 +1,24 @@
+/*
+ * Copyright Camunda Services GmbH and/or licensed to Camunda Services GmbH under
+ * one or more contributor license agreements. See the NOTICE file distributed
+ * with this work for additional information regarding copyright ownership.
+ * Licensed under the Zeebe Community License 1.1. You may not use this file
+ * except in compliance with the Zeebe Community License 1.1.
+ */
+package io.camunda.zeebe.gateway.rest;
+
+import java.lang.annotation.Documented;
+import java.lang.annotation.ElementType;
+import java.lang.annotation.Retention;
+import java.lang.annotation.RetentionPolicy;
+import java.lang.annotation.Target;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+@Retention(RetentionPolicy.RUNTIME)
+@Target({ElementType.TYPE, ElementType.METHOD})
+@Documented
+@ConditionalOnRestGatewayEnabled
+@RestController
+@RequestMapping("/api/v1")
+public @interface ZeebeRestController {}
```

Comment:
:question: Did this make sense to you, using this? Do you think it's premature?

Reply:
As we discussed. this is an internal annotation, and in case a second version of the REST API is created, we can refactor it, or create a new one.